qemu/tests/cfg/virtio_driver_sign_check.cfg

```diff
@@ -1,16 +1,26 @@
 - virtio_driver_sign_check:
     virt_test_type = qemu
-    only Win7.x86_64
-    # This script only need run once for every virtio driver version.
-    # So need not add to our testing loop.
+    only Windows
     type = virtio_driver_sign_check
-    cdroms += " sdk virtio"
-    cdrom_virtio = isos/windows/virtio-win.iso
-    cdrom_sdk = isos/windows/GRMSDKX_EN_DVD.iso
+    cdroms += " virtio"
+    cdrom_virtio = isos/windows/virtio-win.latest_prewhql.iso
+    floppy_readonly = yes
     floppies = "fl"
-    floppy_name = images/win7-64/virtio-win.vfd
-    list_files_cmd = dir /s /b %s | find "%s"
-    winsdk_au3 = autoit/winsdk.au3
-    signtool_install = "D:\autoit3.exe c:\winsdk.au3"
-    signtool_cmd = "c:\Program Files\Microsoft SDKs\Windows\v7.0\Bin\signtool.exe" verify /v /kp /c %s %s
-    drive_list = F: A:
+    list_files_cmd = 'dir /s /b %s | find "%s"'
+    get_disk_letter_cmd = 'wmic logicaldisk | find "virtio-win"'
+    drive_volume = A:
+    tested_drivers = "Balloon pvpanic viorng vioserial vioinput NetKVM netkvm vioscsi viostor"
```

Comment:
Two netkvm drivers' name?
NetKVM netkvm

Reply:
lijin, yes, here need  both netkvm and NetKVM, it's due to the directory name is different on cdrom and floopy, define 2 names to make sure the correct directory can be searched when test no matter driver on cdrom or floopy. only the netkvm is special, so define them here.